content/en/docs/components/katib/env-variables.md

```diff
@@ -5,14 +5,16 @@ weight = 50
                     
 +++
 
-This page describes information about environment variables for each Katib component. If you want to change Katib installation, you can modify some of these variables.
+This page describes information about environment variables for each Katib component.
+If you want to change Katib installation, you can modify some of these variables.
 
-In the tables below you can see description, default values and mandatory property for all environment variables in each Katib component.
-If variable has mandatory property, appropriate Katib component's manifest must have this environment variable.
+In the tables below you can see description, default values and mandatory property for
+all environment variables in each Katib component. If variable has mandatory property,
+appropriate Katib component's manifest must have this environment variable.
```

Comment:
Suggestions:

- Change to plural (because of "all variables" per component)
- Add indefinite articles ("a")

```suggestion
In the tables below there are descriptions, default values and mandatory properties for
all environment variables in each Katib component. If a variable has a mandatory property,
you need to set the relevant environment variable in an appropriate Katib component's manifest.
```

(Please check if the meaning is still the same - we're giving instructions to "you" (the reader), so changing to active voice)

(It may also make sense to link this to 80 char per line)